### DISCUSSION TITLE
Support Mx450?

### DISCUSSION THREAD

**Discussion by Decodetalkers** [NVIDIA/open-gpu-kernel-modules#309] [Q&A] [answered]
Does the new release support Mx450? I have not seen my Graphic gpu..

**Comment by aaronp24** [comment, marked as answer]
The MX450 is a Turing GPU so in theory it should would. I haven't tried it myself, though. Please note, however, that technically only the specific GPUs listed in the ["Compatible GPUs"](https://github.com/NVIDIA/open-gpu-kernel-modules#compatible-gpus) list are actually supported by this version of the driver.
